shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/LocationScreen.kt

```diff
@@ -0,0 +1,35 @@
+package org.jetbrains.kotlinconf.ui
+
+import androidx.compose.foundation.gestures.rememberTransformableState
+import androidx.compose.foundation.gestures.transformable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.graphicsLayer
+import org.jetbrains.kotlinconf.ui.components.TabBar
+
+@Composable
+fun LocationScreen() {
+    var floor by remember { mutableStateOf("FLOOR 0") }
+
+    Box {
+        SvgView(Modifier.fillMaxSize())
+        TabBar(
+            tabs = listOf("FLOOR -1", "FLOOR 0", "FLOOR 1"),
+            selected = floor,
+            onSelect = { floor = it },
+        )
+
+    }
+}
+
+@Composable
+expect fun SvgView(modifier: Modifier)
```

Comment:
I would name it `SvgMapView`(filePath: String, modifier: Modifier)

Reply:
thanks, will fix